suites/rbd/qemu/pool/replicated-data-pool.yaml

```diff
@@ -0,0 +1,21 @@
+tasks:
+- exec:
+    client.0:
+      - sudo ceph osd erasure-code-profile set teuthologyprofile ruleset-failure-domain=osd m=1 k=2
```

Comment:
Minor: was this intentional? This appears to be testing a replicated data pool with an EC-backed cache pool over the rbd pool